dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/saaj/SAAJUtils.java

```diff
@@ -101,13 +101,8 @@ public static Element adjustPrefix(Element e, String prefix) {
                     ((SOAPElement)e).removeNamespaceDeclaration(s);
                 } else if ("com.sun.org.apache.xerces.internal.dom.ElementNSImpl".equals(
                        e.getClass().getName())) {
-                    //since java9 159 SOAPPart1_1Impl.getDocumentElement not return SOAPElement
-                    try {
-                        Method method = e.getClass().getMethod("removeAttribute", String.class);
-                        method.invoke(e, "xmlns:" + s);
-                    } catch (Exception ex) {
-                        ex.printStackTrace();
-                    }
+                    
+					e.removeAttribute("xmlns:" + s); // Liberty Change
```

Comment:
```suggestion
                    //since java9 159 SOAPPart1_1Impl.getDocumentElement not return SOAPElement
                    e.removeAttribute("xmlns:" + s); // Liberty Change
```